<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapLoadModule</class>
 <widget class="QDialog" name="MapLoadModule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>633</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>4</number>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <widget class="QWidget" name="raster_tab">
      <attribute name="icon">
       <iconset resource="../Resources.qrc">
        <normaloff>:/new/prefix1/icons/Layers/rasterLayer.png</normaloff>:/new/prefix1/icons/Layers/rasterLayer.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Raster</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>URL :</string>
           </property>
           <property name="indent">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLineEdit" name="le_rasterUrl"/>
           </item>
           <item>
            <widget class="QPushButton" name="pb_rasterUrl">
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="cb_raster">
         <property name="font">
          <font>
           <italic>true</italic>
          </font>
         </property>
         <property name="text">
          <string>Use file name as default layer name</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <property name="tristate">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Layer Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_rasterLabelName">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="vector_tab">
      <attribute name="icon">
       <iconset resource="../Resources.qrc">
        <normaloff>:/new/prefix1/icons/Layers/vectorLayer.png</normaloff>:/new/prefix1/icons/Layers/vectorLayer.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Vector</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>URL :</string>
           </property>
           <property name="indent">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLineEdit" name="le_vectorUrl"/>
           </item>
           <item>
            <widget class="QPushButton" name="pb_vectorUrl">
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="cb_vectorLayerName">
         <property name="font">
          <font>
           <italic>true</italic>
          </font>
         </property>
         <property name="text">
          <string>Use file name as default layer name</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <property name="tristate">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Layer Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_vectorLayerName"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="elevation_tab">
      <attribute name="icon">
       <iconset>
        <normaloff>icons/Layers/elevationLayerColor.png</normaloff>icons/Layers/elevationLayerColor.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Elevation</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>URL :</string>
           </property>
           <property name="indent">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLineEdit" name="le_eleUrl"/>
           </item>
           <item>
            <widget class="QPushButton" name="pb_eleUrl">
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="cb_eleLayerName">
         <property name="font">
          <font>
           <italic>true</italic>
          </font>
         </property>
         <property name="text">
          <string>Use file name as default layer name</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <property name="tristate">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Layer Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_eleLayerName"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="sceanTab">
      <attribute name="icon">
       <iconset resource="../Resources.qrc">
        <normaloff>:/new/prefix1/icons/Layers/3D Scene Layer.png</normaloff>:/new/prefix1/icons/Layers/3D Scene Layer.png</iconset>
      </attribute>
      <attribute name="title">
       <string>3D Scene</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>URL :</string>
           </property>
           <property name="indent">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLineEdit" name="le_sceanUrl"/>
           </item>
           <item>
            <widget class="QPushButton" name="pb_sceneUrl">
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Height Attribute : </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cb_heightAttribute"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Resource File :(eg: .xml)</string>
           </property>
           <property name="indent">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLineEdit" name="le_sceneXml">
             <property name="text">
              <string>D:/OSG1/osgEarth/data/resources/textures_us/catalog.xml</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pb_sceneXml">
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="cb_scenelayerName">
         <property name="font">
          <font>
           <italic>true</italic>
          </font>
         </property>
         <property name="text">
          <string>Use file name as default layer name</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <property name="tristate">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Layer Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_sceneLayerName"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="icon">
       <iconset resource="../Resources.qrc">
        <normaloff>:/new/prefix1/icons/Layers/XYZ.png</normaloff>:/new/prefix1/icons/Layers/XYZ.png</iconset>
      </attribute>
      <attribute name="title">
       <string>XYZ</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_12">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QRadioButton" name="fromServer_radioButton">
           <property name="text">
            <string>From Server</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="fromLocal_radioButton">
           <property name="text">
            <string>From Local</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_11">
         <item>
          <widget class="QLabel" name="xyzUrlLabel">
           <property name="text">
            <string>URL :</string>
           </property>
           <property name="indent">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QComboBox" name="cb_ServerUrl">
             <property name="editable">
              <bool>true</bool>
             </property>
             <item>
              <property name="text">
               <string>https://tile.openstreetmap.org/{z}/{x}/{y}.png</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>http://tile.thunderforest.com/cycle/{z}/{x}/{y}.png</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>http://mt0.google.com/vt/lyrs=m&amp;hl=en&amp;x={x}&amp;y={y}&amp;z={z}</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>http://mt0.google.com/vt/lyrs=p&amp;hl=en&amp;x={x}&amp;y={y}&amp;z={z}</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>http://mt0.google.com/vt/lyrs=r&amp;hl=en&amp;x={x}&amp;y={y}&amp;z={z}</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>http://mt0.google.com/vt/lyrs=s&amp;hl=en&amp;x={x}&amp;y={y}&amp;z={z}</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>http://mt0.google.com/vt/lyrs=t&amp;hl=en&amp;x={x}&amp;y={y}&amp;z={z}</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>http://mt0.google.com/vt/lyrs=y&amp;hl=en&amp;x={x}&amp;y={y}&amp;z={z}</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>https://ecn.t3.tiles.virtualearth.net/tiles/a{quadkey}.jpeg?g=1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>https://cartodb-basemaps-a.global.ssl.fastly.net/light_all/{z}/{x}/{y}.png</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>https://stamen-tiles.a.ssl.fastly.net/watercolor/{z}/{x}/{y}.jpg</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="le_xyzUrl">
             <property name="text">
              <string>http://[abc].tile.openstreetmap.org/{z}/{x}/{y}.png</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pb_xyzUrl">
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="cb_xyzLayerName">
         <property name="font">
          <font>
           <italic>true</italic>
          </font>
         </property>
         <property name="text">
          <string>Use file name as default layer name</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <property name="tristate">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Layer Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_xyzLayername">
         <property name="text">
          <string>osm</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="warning_label">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pb_openMap">
       <property name="text">
        <string>Open</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Resources.qrc"/>
 </resources>
 <connections/>
</ui>
